kytos/core/auth.py

```diff
@@ -109,21 +216,12 @@ def get_email():
             re_password = getpass.getpass('Retype password: ')
             if password == re_password:
                 break
-            print('Passwords do not match. Try again')
-
         user = {
             "username": username,
             "email": email,
-            "password": hashlib.sha512(password.encode()).hexdigest(),
+            "password": password,
         }
-        content = {
-            "namespace": self.namespace,
```

Comment:
Feel free to nuke this `self.namespace` that's still being initialized in the constructor too.

Reply:
I didn't notice it, removed.